src/Simplify.elm

```diff
@@ -8783,6 +8765,15 @@ subCollection =
     }
 
 
+{-| The map checks
+
+    map f empty --> empty
+
+    map identity emptiable --> emptiable
+
+If your mapping function also takes extra information like the key or index as an argument, use `emptiableMapWithExtraArgChecks`.
```

Comment:
Just wanted to say that I think that these documentation comments are really valuable, this is great :heart: